src/Model/EloquentObserver.php

```diff
@@ -87,6 +88,10 @@ public function updatedMultiple(EloquentModel $model)
      */
     public function deleting(EloquentModel $entry)
     {
+        if (!$this->dispatch(new RestrictDelete($entry))) {
```

Comment:
Maybe it is better to use comparing with true here and return inverted from the command?

Reply:
Sound good :)